### DISCUSSION TITLE
Format avif not support in Astro?

### DISCUSSION THREAD

**Discussion by layout-ch** [ascorbic/unpic-img#696] [General]
Hey !

In the doc its says that the unpic Image component has more attributes than the Astro Image component. But the format attribute seems missing which is crucial for performances....

Am I doing something wrong or is this really not supported?

**Comment by jstask82** [comment]
I consider unpic-img for my project atm. Does the attribute type="image/avif" as described in https://dev.to/ascorbic/level-up-your-web-images-with-unpic-picture-tags-4jh1 work for you?
